src/Objs/defaultPageAttributes.ts

```diff
@@ -4,7 +4,7 @@ export const defaultPageAttributes = {
   // Layout widgets
   layoutHeader: 'widgetlist',
   layoutShowHeader: 'boolean',
-  layoutFooter: ['widgetlist', { only: 'SectionWidget' }],
+  layoutFooter: ['widgetlist', { only: ['FooterWidget', 'SectionWidget'] }],
```

Comment:
* [ ] Wouldn't a FooterWidget in the layoutFooter (of the homepage) lead to an endless recursion?
* [ ] FYI I'm not sure about the intended use. From what I can see, the new widget only makes sense on a standalone page with no path. For a standalone page, the layoutFooter makes not much sense, as putting a footer at the end of the main content has the same effect. If my guess about the intended use is correct, we could repurpose the existing `layoutShowFooter` to show the homepage footer instead of introducing a widget for that.